models/tmp/stg_greenhouse__user_tmp.sql

```diff
@@ -1 +1 @@
-select * from {{ var('user') }}
+select * from {{ var('user') if does_table_exist('user') else var('users') }}
```

Comment:
```suggestion
select * from {{ var('users') if var('greenhouse__using_users', does_table_exist('users')) else var('user') }}
```
Same here to priortize "users" and add the var.